src/components/plugins/PluginDialog.tsx

```diff
@@ -0,0 +1,45 @@
+import { ReactElement, Dispatch, SetStateAction } from "react";
+import { Dialog, Paper } from "@mui/material";
+import makeStyles from "@mui/styles/makeStyles";
+import { theme, IconButton, icons } from "@gliff-ai/style";
+
+interface Props {
+  children: JSX.Element | null;
+  setChildren: Dispatch<SetStateAction<JSX.Element>>;
+}
+
+const useStyle = makeStyles({
+  dialogPaper: {
+    width: "100%",
+    height: "70%",
+    backgroundColor: theme.palette.primary.light,
+  },
+  paper: {
+    width: "100%",
+    height: "100%",
+  },
+  button: {
+    float: "right",
+  },
+});
```

Comment:
No useStyle if possible, you'll make @ChrisBaidoo sad when he has to remove it 😬 

(You'd better check with him what the correct alternative is, but for these simple styles, inline `style={}` should be fine?)

(same comment for other make/usestyle uses)

Reply:
ok, replaced. For future reference I had to add a state for the on hover styles, because "&:hover" doesn't work with inline style.

Reply:
Hmm there should be a better way...but I don't know it and this will do for now!